src/plugins/ecs/__init__.py

```diff
@@ -452,7 +462,7 @@ def information():
 	jarvis ecs describe|desc <cluster> [in <region/account>]
 	jarvis ecs describe|desc <service> <cluster> [in <region/account>]
 	jarvis ecs list tasks[---<task_name_optional>] running <cluster> [in <region/account>]
-	jarvis ecs compare <cluster> [in <region> <account>] with <cluster> [in <region> <account>]"""
+	jarvis ecs compare [<cluster>] into <region> <account> with [<cluster>] into <region> <account>"""
```

Comment:
yes I found that using "in" would conflict with the other commands in jarvis, it leads to jarvis taking in the last region from one of the comparables. Using something besides "in" sidesteps jarvis parsing out the two comparables